ngraph.mk

```diff
@@ -145,40 +165,72 @@ $(NGRAPH_BRIDGE_OBJ): %.o: ngraph $(NGRAPH_BRIDGE_SRC)
   # nGraph provides some libraries that may compete with other libraries already installed
   # on the system. This provides the link-flags that, if provided early enough on the static-linker
   # command line, should ensure that the nGraph-supplied version is preferred.
+  ifeq ("$(shell uname)","Darwin")
+    NGRAPH_MKLML_LIB_ := mklml
```

Comment:
Does MKLDNN fall back to libmklml on Mac?

Reply:
Short answer: I'm not sure.

tl;dr:  I was planning to tackle this question in a follow-on PR, but I didn't want to hold things up.  IIRC this change was prompted by a combination of failed builds and of seeing the logic [in this file](https://github.com/NervanaSystems/ngraph-mxnet/blob/master/cmake/MklDnn.cmake).

Reply:
@mbrookhart : A partial answer to your question: [https://mxnet.apache.org/install/build_from_source.html#math-library-selection](https://mxnet.apache.org/install/build_from_source.html#math-library-selection).